AnkiDroid/src/main/java/com/ichi2/anki/IntentHandler.kt

```diff
@@ -156,6 +156,11 @@ class IntentHandler : Activity() {
 
     private fun handleFileImport(intent: Intent, reloadIntent: Intent, action: String?) {
         Timber.i("Handling file import")
+        if (!hasShownAppIntro()) {
+            Timber.i("Trying to import a file when the app was not started at all")
+            showThemedToast(this, R.string.warning_start_app_before_import, false)
```

Comment:
I prefer a more generic string name here that captures the "why" not the "what"/"how"

```suggestion
            showThemedToast(this, R.string.app_not_initialized, false)
```